ros2_control_demo_robot/description/rrbot/rrbot_system_with_sensor.ros2_control.xacro

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0"?>
+<robot xmlns:xacro="http://www.ros.org/wiki/xacro">
+
+  <xacro:macro name="rrbot_system_with_sensor" params="name prefix">
+
+    <ros2_control name="${name}" type="system">
+      <hardware>
+        <plugin>ros2_control_demo_hardware/RRBotSystemWithSensorHardware</plugin>
+        <param name="example_param_write_for_sec">2</param>
+        <param name="example_param_read_for_sec">2</param>
+      </hardware>
+      <joint name="joint1">
+        <command_interface name="position">
+          <param name="min">-1</param>
+          <param name="max">1</param>
+        </command_interface>
+        <state_interface name="position"/>
+      </joint>
+      <joint name="joint2">
+        <command_interface name="position">
+          <param name="min">-1</param>
+          <param name="max">1</param>
+        </command_interface>
+        <state_interface name="position"/>
+      </joint>
+      <sensor name="tcp_fts_sensor">
+        <state_interface name="fx"/>
+        <state_interface name="fy"/>
+        <state_interface name="fz"/>
+        <state_interface name="tx"/>
+        <state_interface name="ty"/>
+        <state_interface name="tz"/>
+        <param name="frame_id">kuka_tcp</param>
+        <param name="lower_limits">-100</param>
+        <param name="upper_limits">100</param>
```

Comment:
Propose to update this as in ros-controls/roadmap#32. This simplifies the example. There is no need to have 6-axis sensor.

```suggestion
      <state_interface name="fx"/>
      <state_interface name="tz"/>
      <param name="frame_id">rrbot_tcp</param>
      <param name="fx_range">100</param>
      <param name="tz_range">15</param>
```